client/elements/text/sc-simple-text.html

```diff
@@ -305,13 +305,16 @@
               if (!this.sutta || !this.sutta.uid) {
                   return;
               }
-              const divisionId = /^[a-z]+/.exec(this.sutta.uid)[0];
+              let divisionId = /^[a-z]+/.exec(this.sutta.uid)[0];
+              if (divisionId === 'pli') {divisionId = 'vi'};
+              if (divisionId === 'iti') {divisionId = 'it'};
               if (this.paragraphs && this.showParagraphs) {
                   this.paragraphs.forEach((paragraph) => {
                       const refs = this.$.simple_text_content.querySelectorAll(`.${paragraph.uid}`);
                       Array.from(refs).forEach((item) => {
-                          if (item.id.startsWith('pts')) {
-                              this._processVolPageInfo(item, divisionId);
+                          if (divisionId !== 'tha' && divisionId !== 'thi' && !item.id.includes('2ed')
```

Comment:
This if condition is awfully complicated, I'd extract it to a separate function with a clear name.

Reply:
Yup, this stuff is so complicated with many exceptions and you cannot find a common ground between them so you have to get all this in. Will make it into a separate function.